papers/vyas_ramasubramani/94_vyas_ramasubramani.rst

```diff
@@ -73,15 +71,15 @@ A simple solution might be to just rename the files to account for this paramete
 A more flexible traditional solution involving the use of a relational MySQL :cite:`mysql` database, for instance, might introduce undesirable setup costs and performance bottlenecks for file-based workflows on HPC.
 Even if we do employ such a solution, we also have to account for our workflow process: we need a way to run analysis and post-processing on just the new data points without performing unnecessary work on the old ones.
 
-This paper showcases the ``signac`` framework, a data and workflow management tool that aims to address these issues in a simple, powerful, and flexible manner (Fig. :ref:`fig:summary`).
-By storing JSON-encoded metadata and the associated data together directly on the file system, ``signac`` provides database functionality such as searching and grouping data without the overhead of maintaining a server or interfacing with external systems, and it takes advantage of the high performance file systems common to HPC.
+This paper showcases the ``signac`` framework, a data and workflow management tool that addresses these issues in a simple, powerful, and flexible manner (Fig. :ref:`fig:summary`).
+The framework derives its name from the painter Paul Signac, one of the early pioneers of the Pointillist painting style.
```

Comment:
That sentence is now repetitive. Maybe add a note above that the name will be explained later (here)?

Reply:
Are you referring to the abstract?

Reply:
Yes, I am. Didn't realize that. Never mind, then.